sigs.yml

```
@@ -696,3 +696,25 @@
     gcLiaison:
       - name: 'Juraci Paixão Kröhling'
         github: jpkrohling
+  - name: Developer Experience
+    meeting: Every Wednesday 9:00 and 18:00 UTC
```

```suggestion
    meeting: Wednesday 11:00 PT and 17:00 UTC+8
```

Please see #2237 and #2249, make sure you set your times accordingly